### DISCUSSION TITLE
Spreading the theme using require gets typescript complaining. Is there any downside to using this approach instead?

### DISCUSSION THREAD

**Discussion by PaulHaze** [saadeghi/daisyui#3103] [General]
So when customizing a theme the docs say to use this syntax:
`...require("daisyui/src/theming/themes")["light"],`

However my lint rules have a moan about that and I need to end up disabling a few in my tailwind.config.ts

Whilst having a hack around with this I'm now using this approach:

```
import themes from 'daisyui/src/theming/themes';
...

  light: {
    ...themes.light,
    // add other colours and stuff here
  }
```

Is this going to cause problems down the line? At this stage it all seems to work fine and I am getting no complaints, but I'm still quite a novice when it comes to coding so thought I would quickly check in here and see if I've messed up....

**Comment by saadeghi** [comment]
It's the same thing basically. Nothing to worry about
